### PR TITLE
feat(List Patches): Added Page to List Patches. 

### PR DESCRIPTION
**Pull request by deeptea22**
### Functionality:
A page to display all the patches.

### Solution:
Added 3 components to list patches.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start
move to /ListPatches

### CODE REVIEW

**Comment by deeptea22** [comment]
Have resolved all the changes requested in the previous pull request.

File: functions/model/patch/editPatch.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { updatePatchData, getPatchData } = require("./lib");
+
+exports.editPatch = function(request, response) {
+    const Name = request.body.data.Name;
+    const Description = request.body.data.Description;
+    const PatchId = request.body.data.Id;
```
Comment (arjund-06) [inline functions/model/patch/editPatch.js:6]: It would be better to use camelCasing

File: functions/model/patch/patch.js [hunk 2]
```diff
@@ -31,6 +32,9 @@ exports.patch = functions.https.onRequest((request, response) => {
             return patch4(request, response);
         } else if (mode == "patchModerator") {
             return patchModerator(request, response);
+        } else if (mode == "edit") {
+            console.log("edit");
```
Comment (arjund-06) [inline functions/model/patch/patch.js:36]: Remove this log

File: src/app/list-patches/edit-patch/edit-patch.component.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    try{
+      const result = await callable({mode: "edit",Id: this.patch,Name:this.editPatch.Name, Description:this.editPatch.Description}).toPromise();
+      console.log(result);
+      this.enableLoader = false;
+      this.showClose = true;
+    }
+    catch(error){
+      console.log("Error: "+error);
+    }
+  }
```
Comment (arjund-06) [inline src/app/list-patches/edit-patch/edit-patch.component.ts:49]: This can be one single function I guess

**Review by arjund-06 — CHANGES_REQUESTED** [review]

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**